massgov/openmass
# DP-32245: Modify session_parent_orgs to include node ids

**Description:**
See ticket


**Jira:** (Skip unless you are MA staff)
DP-32245


**To Test:**

- Navigate to one or more pages with related orgs, such as `/orgs/executive-office-of-elder-affairs`
- Open the console
- Go to the application tab
- View the value for the session_parent_orgs key under session storage
- It should be in the format of firstOrg:123,secondOrg:124


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
